sdk/samples/iot/aziot_esp32/how_to_iot_hub_esp32.md

```diff
@@ -1,3 +1,15 @@
+---
+page_type: sample
```

Comment:
Don't all these md files need to be renamed to readme.md ?

Reply:
sure thing. updated on the 2nd commit.